docs/setup-dotnet-net-core.asciidoc

```diff
@@ -91,9 +91,9 @@ public static IHostBuilder CreateHostBuilder(string[] args) =>
 
 [float]
 [[zero-code-change-setup]]
-==== Zero code change setup on .NET Core (added[1.7])
+==== Zero code change setup on .NET Core (added[1.7]) and .NET 5+
 
-If you can't or don't want to reference NuGet packages in your application, you can use the startup hook feature to inject the agent during startup, if your application runs on .NET Core. This feature is supported on .NET Core 3.0 and newer versions.
+If you can't or don't want to reference NuGet packages in your application, you can use the startup hook feature to inject the agent during startup, if your application runs on .NET Core or .NET 5 or newer. This feature is supported on .NET Core 3.0 and newer versions.

```

Comment:
The last sentence seems redundant.
What about "runs on .NET Core 3.0 and .NET 5 or newer"?

Reply:
👍 yeah, sounds good.

If I remember correctly, the intention was to emphasize that older versions (like .NET Core 2.x) don't support this feature. But those versions are out-of-support since a long time ago, plus the suggestion covers the exact version anyway.